src/components/hall-of-fame-table-component.js

```diff
@@ -0,0 +1,88 @@
+import React from "react"
+
+export default function HallOfFameTableComponent({
+  person: { name, handle, handleLink, highestRating, level },
+}) {
+  const colors = {
+    Pupil: "text-green-600",
+    Master: "text-amber-400",
+    Grandmaster: "text-red-500",
+    Expert: "text-blue-700",
+    Specialist: "text-[#66CDAA]",
+    CandidateMaster: "text-fuchsia-600",
+  }
+
+  const codechefColors = {
+    2: "#008000",
+    3: "blue",
+    4: "purple",
+    5: "yellow",
+    6: "#FFA500",
+    7: "#800000",
+  }
+
+  const codechefColors2 = {
```

Comment:
Why two?

Reply:
One is for coloring the star and another one is for text color. I tried adding colors to text but those are not working properly so i took another one for coloring the text